web/env.html

```diff
@@ -51,20 +51,16 @@
             <a id="newButton" class="cw-button red">
               <i class="mdi mdi-18px mdi-plus"></i>&nbsp; New
             </a>
-            <a id="saveButton" class="cw-button blue" style="display: none;">
```

Comment:
I really wish these style nits didn't keep polluting pull requests.  Isn't this why we have formatting tools?  It would be far easier to review your pull request if you had not decided to delete all the trailing semicolons. :(  Can you move it to a different pull request or something, at least?

I'm going to wait for something to change, because I don't want to miss something important as I'm skipping the fidgets with formatting.

Reply:
Looks like you snuck these into a merge commit on a different pull request as well... so I guess it's done, now, whether I wanted it or not.  Just merge or something so this is easier to review.

I don't understand why you're trying so hard to remove all the trailing semicolons anyway.  We've discussed this in the past, and decided to keep them, didn't we?

Reply:
Okay, never mind.  It's a waste of time fighting you on it.  Do whatever.  I'll find a formatting tool to enforce a standard when I get around to it.

Reply:
That wasn't me, Chris. All commited files are automatically linted and reprinted, remember? When I decide to refactor something, I usually go for something more meaningful than removing semicolons ...

I clearly marked two commits in this PR as refactors. Do I understand correctly that you would've preferred those to be sent as a standalone PR?

Reply:
Okay, understood.  Sorry I got frustrated.  It's strange that the formatter suddenly changed its mind here when it never did that before... but okay.

Generally speaking, it is much easier if you send refactoring as a separate pull request.  Separating them into commits doesn't actually help, because I'm reviewing all changes from the pull request together.

Reply:
> It's strange that the formatter suddenly changed its mind here when it never did that before

We are using Prettier's latest version at the moment of committing changes, and that implies that it might behave differently from time to time. We can pin the version, if you like.